packages/client/src/components/manage-users/return-books-confirm-dialog.vue

```diff
@@ -36,12 +36,33 @@
       <span class="bg-grey-1 height-48 q-pa-md text-size-13 text-weight-medium">
         {{ tableTitle }}
       </span>
+
       <q-separator />
+
       <dialog-table
         :columns="columns"
         :rows="booksToReturn"
-        :rows-per-page-options="[0]"
-      />
+        class="flex-delegate-height-management"
+      >
+        <template #body-cell-author="{ value, col }">
+          <q-td :class="col.classes">
+            <q-tooltip>
+              {{ value }}
+            </q-tooltip>
+            {{ value }}
+          </q-td>
```

Comment:
This snippet is repeated 10 times, so it would be a good idea to make it a component.